mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -1274,6 +1293,10 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
                         }}
                         title="Agent can create plans and run code."
                     />
+                    <ModelSelector onConfigChange={(config) => {
```

Comment:
Instead of evenly spacing this, I think we can just always keep it on the left next to the Chat/Agent selection code
<img width="687" alt="Screenshot 2025-05-27 at 3 11 52 PM" src="https://github.com/user-attachments/assets/c3aea2f3-0046-43b1-a86d-8c46d4de62c6" />